app/Models/Asset.php

```diff
@@ -221,6 +221,10 @@ public function save(array $params = [])
                     if($field->format == 'BOOLEAN'){
                         $this->{$field->db_column} = filter_var($this->{$field->db_column}, FILTER_VALIDATE_BOOLEAN);
                     }
+                    // Encrypted Fields are validated in the controller, we remove the validation rule here
+                    if ($field->field_encrypted == 1){
+                        unset($this->rules[$field->db_column]);
+                    }
```

Comment:
Is it possible to hit this method from somewhere other than the `AssetsController` that would throw this off? Maybe during the import process?

Reply:
That's an excellent point, Marcus. 👍

Reply:
I'm already did the API, but in the importer I feel a little bit lost. So If friends can look at what I'm doing in the AssetImporter, please. It already validates correctly, just having a hard time returning the error to the importer GUI. But it's late and I want to know if the advance is solid to finish this tomorrow

Reply:
I think showing the error is going to be affected by the work that @uberbrady did by moving the front end to Livewire. I think it'll be a good idea to pull develop into this branch before debugging further.

I did that locally (I'll let you do it locally too so I don't push unexpected commits to your branch) and took a brief look into it. I'm going to avoid going too much further and ask @uberbrady if he has any helpful insight or ideas on this?